setuptools/__init__.py

```diff
@@ -13,6 +13,7 @@
 
 from ._deprecation_warning import SetuptoolsDeprecationWarning
 
+import _setuptools_vendored  # noqa: F401
```

Comment:
I recommend adding a code comment with an explanation of what this does and why it's important to keep it in this specific position.